docs/developers/deploy-from-a-branch.rst

```diff
@@ -100,6 +100,7 @@ Make your changes, then click :guilabel:`Save`.
 Normally, these changes will immediately take effect.
 Sometimes if you change a ``ConfigMap`` you will need to restart the relevant deployments to pick up that change.
 For instructions on how to do that, see :ref:`branch-deploy-restart`.
+(However, see :ref:`dev-deployment-restart` for a better way.)
```

Comment:
```suggestion
(However, see :ref:`dev-deployment-restart` for a better way.)
To configure the chart to make these restarts happen automatically, see :ref:`dev-deployment-restart`.
```

Reply:
Yes, this is better, but when applying it, I realized that my whole original change was wrong. The checksum is only computed during Helm operations, and this section is on making changes directly in Argo CD, so the restart always has to be done manually. I'll revise it a different way to make that clear.